src/highdicom/pm/enum.py

```diff
@@ -0,0 +1,45 @@
+"""Enumerate values specific to Parametric Map IODs."""
+from enum import Enum
+
+
+class ImageFlavorValues(Enum):
```

Comment:
Should we consider including the [CT- and MR-specific defined terms from C.8.15.2.1.1.3 and C.8.13.1.1.1.3](https://dicom.innolitics.com/ciods/parametric-map/parametric-map-image/00080008)? (also for the derived pixel contrast values)

Reply:
Addressed via 7216af5c10b05131f42f9ec1a5462f0ede2ebe1f